PAS_Database/dbo/Stored Procedures/GetReceivingPurchaseOrderViewById.sql

```diff
@@ -0,0 +1,425 @@
+		LEFT JOIN  [dbo].[StockLineDraftManagementStructureDetails] SLDM WITH(NOLOCK) ON SLDM.ReferenceID = SLD.StockLineDraftId AND ModuleID = 31
+		LEFT JOIN  [dbo].[ShippingVia] SV WITH(NOLOCK) ON SV.ShippingViaId = SLD.ShippingViaId
+		LEFT JOIN  [dbo].[GLAccount] GL WITH(NOLOCK) ON GL.GLAccountId = SLD.GLAccountId
+		LEFT JOIN  [dbo].[Condition] C WITH(NOLOCK) ON C.ConditionId = SLD.ConditionId
+		LEFT JOIN  [dbo].[ItemMaster] IMN WITH(NOLOCK) ON IMN.ItemMasterId = SLD.NHAItemMasterId
+		LEFT JOIN  [dbo].[ItemMaster] IMT WITH(NOLOCK) ON IMT.ItemMasterId = SLD.TLAItemMasterId
+		LEFT JOIN  [dbo].[Site] S WITH(NOLOCK) ON S.SiteId = SLD.SiteId
+		LEFT JOIN  [dbo].[Warehouse] W WITH(NOLOCK) ON W.WarehouseId = SLD.WarehouseId
+		LEFT JOIN  [dbo].[Location] L WITH(NOLOCK) ON L.LocationId = SLD.LocationId
+		LEFT JOIN  [dbo].[Shelf] SH WITH(NOLOCK) ON SH.ShelfId = SLD.ShelfId
+		LEFT JOIN  [dbo].[Bin] B WITH(NOLOCK) ON B.BinId = SLD.BinId
+		WHERE SLD.PurchaseOrderId=@PurchaseOrderId
+
+
+		/* START SELECT FROM NonStockInventoryDraft */
+		SELECT 
+		SLD.PurchaseOrderPartRecordId,
+		SLDM.LastMSLevel,
+		SLDM.AllMSlevels,
+		SLD.NonStockInventoryDraftId AS 'StockLineDraftId',
+		SLD.NonStockInventoryNumber AS 'StockLineNumber',
+		SLD.ControlNumber,
+		SLD.IdNumber,
+		SLD.ConditionId,
+		SLD.SerialNumber,
+		SLD.Quantity,
+		SLD.UnitCost AS 'PurchaseOrderUnitCost',
+		SLD.ExtendedCost AS 'PurchaseOrderExtendedCost',
+		SLD.ReceiverNumber,
+		0 AS WorkOrder,
+		0 AS SalesOrder,
+		0 AS SubWorkOrder,
+		'' AS 'OwnerType',
+		'' AS 'ObtainFromType',
+		'' AS 'TraceableToType',
+		'' AS 'ManufacturingTrace',
+		SLD.ManufacturerId,
+		SLD.Manufacturer,
+		'' AS 'ManufacturerLotNumber',
+		'' AS 'ManufacturingDate',
+		'' AS 'ManufacturingBatchNumber',
+		'' AS 'PartCertificationNumber',
+		'' AS 'EngineSerialNumber',
+		SLD.ShippingViaId,
+		SLD.ShippingVia,
+		SLD.ShippingReference,
+		SLD.ShippingAccount,
+		'' AS 'CertifiedDate',
+		'' AS 'CertifiedBy',
+		'' AS 'TagType',
+		'' AS 'TagDate',
+		SLD.MfgExpirationDate,
+		'' AS 'CertifiedDueDate',
+		'' AS 'AircraftTailNumber',
+		SLD.GLAccountId,
+		SLD.GLAccount,
+		SLD.Condition,
+		SLD.ManagementStructureId,
+		SLD.SiteId,
+		SLD.WarehouseId,
+		SLD.LocationId,
+		SLD.ShelfId,
+		SLD.BinId,
+		'' AS 'NHAItemMasterId',
+		'' AS 'TLAItemMasterId',
+		'' AS 'NHAItemMasterText',
+		'' AS 'TLAItemMasterText',
+		SLD.Site AS 'SiteText',
+		SLD.Warehouse AS 'WarehouseText',
+		SLD.Location AS 'LocationText',
+		SLD.Shelf AS 'ShelfText',
+		SLD.Bin AS 'BinText',
+		'' AS 'ObtainFrom',
+		'' AS 'Owner',
+		'' AS 'TraceableTo',
+		SLD.Level1,
+		SLD.Level2,
+		SLD.Level3,
+		SLD.Level4,
+		SLD.UnitOfMeasure,
+		'' AS 'TaggedByName',
+		'' AS 'CertTypeId',
+		'' AS 'CertType'
+		FROM DBO.[NonStockInventoryDraft] SLD WITH (NOLOCK)
+		LEFT JOIN  [dbo].[StockLineDraftManagementStructureDetails] SLDM WITH(NOLOCK) ON SLDM.ReferenceID = SLD.NonStockInventoryDraftId AND ModuleID = 55
```

Comment:
Do not use static moduleId.. Get it by select statement on top and make use of it..